scout/server/blueprints/variants/forms.py

```diff
@@ -145,7 +145,9 @@ class CancerFiltersForm(VariantFiltersForm):
     tumor_frequency = BetterDecimalField(
         "Tumor alt AF >", places=2, validators=[validators.Optional()]
     )
-    mvl_tag = BooleanField("In Managed Variant List")
+    clinvar_tag = BooleanField("ClinVar hits")
+    cosmic_tag = BooleanField("Cosmic hits")
+    mvl_tag = BooleanField("Managed Variants hits")
```

Comment:
I don't know about the naming for these 3, perhaps this was better as it was

Reply:
Potato potato!